uPortal-webapp/src/main/webapp/WEB-INF/flows/edit-portlet/portletConfig.jsp

```diff
@@ -194,7 +194,7 @@ PORTLET DEVELOPMENT STANDARDS AND GUIDELINES
                     </p>
                 </div>
             </c:if>
-            <div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
```

Comment:
What's the story with this one?

Reply:
Using the [Axe](https://www.deque.com/products/axe/) add-on for Firefox it said that `aria-multiselectable` was not an appropriate attribute for the div element. So that's why I removed it.

Reply:
That makes complete sense.  It would be a great remark to include in a JIRA ticket.  ;-)